scriptworker/cot/verify.py

```diff
@@ -960,111 +916,9 @@ def verify_link_in_task_graph(chain, decision_link, task_link):
         verify_task_in_task_graph(task_link, graph_defn)
         log.info("Found {} in the graph; it's a match".format(task_link.task_id))
         return
-    # Fall back to fuzzy matching to support retriggers: the taskId and
```

Comment:
I'm not sure to follow how retriggers work in this model. Does the retrigger action task creates a new task-graph.json and CoT only relies on it?

Reply:
That's the theory, and it works for relpro tasks, which create `task-graph.json` with an accurate `taskId` and task definition.

Retriggers seem to create a `task-graph-0.json`, which isn't supported by CoT verification... the reasoning seems to be that retriggers allow for multiple graphs (e.g. retrigger graph 0-5). This means CoT verification will fail.

However, fuzzy task matching only worked for a single task on current retriggers. Any subsequent tasks failed, e.g. a retriggered signing task might pass CoT verification, but a child beetmover task would fail. Fuzzy task matching adds time, log spam, and doesn't fully solve the problem, so we're leaning towards saying "retriggers aren't compatible with CoT", or "retriggers should be for tests only".